app/models/job.rb

```diff
@@ -0,0 +1,9 @@
+class Job < ApplicationRecord
+  belongs_to :company
+
+  has_rich_text :description
+
+  enum job_type: { 'Full-time' => 0, 'Part-time' => 1 }
```

Comment:
the hash-rockets format (`=>`) is an older format. Let's use the newer version of `:`. Also this would be the best way to do enums:

`enum localization: { home: 0, foreign: 1, none: 2 }`

Where the names are keys and not strings.